frontend/website/src/components/chatroom.svelte

```diff
@@ -100,10 +159,17 @@
 		<span class="chat-title">Chat</span>
 	</div>
 	<div class="messages" bind:this={messagesEl}>
-		{#each messages as message (message.svelte_id)}
-			<div class="message" class:odd={message.svelte_id % 2}>
-				<span class="message-sender" style={`color: ${message.color}`}>{message.sender}:</span>
-				<span class="message-text">{message.text}</span>
+		{#each messages as message}
+			<div class="message">
```

Comment:
No even odd messages?

Reply:
I thought different style for odd/even messages looked worse? I can add it back if you want, maybe with different background color